rfq/manager.go

```diff
@@ -756,6 +756,57 @@ func (q *PeerAcceptedBuyQuoteEvent) Timestamp() time.Time {
 // interface.
 var _ fn.Event = (*PeerAcceptedBuyQuoteEvent)(nil)
 
+// QuoteRespStatus is an enumeration of possible quote response statuses.
+type QuoteRespStatus uint8
+
+const (
+	// InvalidRateTickQuoteRespStatus indicates that the rate tick in the
+	// quote response is invalid.
+	InvalidRateTickQuoteRespStatus QuoteRespStatus = iota
+
+	// InvalidExpiryQuoteRespStatus indicates that the expiry in the quote
+	// response is invalid.
+	InvalidExpiryQuoteRespStatus
+
+	// PriceOracleQueryErrQuoteRespStatus indicates that an error occurred
+	// when querying the price oracle whilst evaluating the quote response.
+	PriceOracleQueryErrQuoteRespStatus
+)
+
+// InvalidQuoteRespEvent is an event that is broadcast when the RFQ manager
+// receives an unacceptable quote response message from a peer.
+type InvalidQuoteRespEvent struct {
+	// timestamp is the event creation UTC timestamp.
+	timestamp time.Time
+
+	// QuoteResponse is the quote response received from the peer which was
+	// deemed invalid.
+	QuoteResponse rfqmsg.QuoteResponse
+
+	// Status is the status of the quote response.
+	Status QuoteRespStatus
```

Comment:
See my other comment regarding having a free text or error message here too to give some additional context to the user. But non-blocking, just something for the wish list to improve user experience.

What things look like from my tests for example:
```
[litcli] peer 02ad73dc4471e83179651add7096d238920fefe58ab721b31cc8a25ea575ffe324 sent back an invalid quote, status: INVALID_RATE_TICK
```

Would be nice if we could also display the message that's currently only in the logs (e.g. "Buy accept price is not within acceptable bounds").
But again, non-blocking.